lib/loopback.js

```diff
@@ -158,7 +158,9 @@ if (loopback.isServer) {
     });
 
   loopback.urlNotFound = loopback['url-not-found'];
+  loopback.errorHandler = loopback['error-handler'];
   delete loopback['url-not-found'];
+  delete loopback['error-handler'];
```

Comment:
Nitpick: group by property name, not by operations.

```
loopback.urlNotFound = loopback['url-not-found'];
delete loopback['url-not-found'];

loopback.errorHandler = loopback['error-handler'];
delete loopback['error-handler'];
```

In this particular case, I'd personally keep `loopback[error-handler]` so that it can be loaded in middleware.json via `loopback#error-handler`. Not a big deal though.